spacepy/irbempy/__init__.py

```diff
@@ -83,11 +83,2472 @@
+        - Dst: Dst index (nT)
+        - dens: Solar Wind density (cm-3)
+        - velo: Solar Wind velocity (km/s)
+        - Pdyn: Solar Wind dynamic pressure (nPa)
+        - ByIMF: GSM y component of IMF mag. field (nT)
+        - BzIMF: GSM z component of IMF mag. field (nT)
+        - G1:  G1=< Vsw*(Bperp/40)2/(1+Bperp/40)*sin3(theta/2) > where the <> mean an average over the
+            previous 1 hour, Vsw is the solar wind speed, Bperp is the transverse IMF
+            component (GSM) and theta it's clock angle.
+        - G2: G2=< a*Vsw*Bs > where the <> mean an average over the previous 1 hour,
+                Vsw is solar wind speed, Bs=|IMF Bz| when IMF Bz < 0 and Bs=0 when IMF Bz > 0, a=0.005.
+        - G3:  G3=< Vsw*Dsw*Bs /2000.> where the <> mean an average over the previous 1 hour,
+                Vsw is the solar wind speed, Dsw is the solar wind density, Bs=|IMF Bz| when IMF
+                Bz < 0 and Bs=0 when IMF Bz > 0.
+        - W1 - W6: see definition in (JGR-A, v.110(A3), 2005.) (PDF 1.2MB)
+        - AL: the auroral index
+
+    Options
+        - 1st element: 0 - don't compute L* or phi ;  1 - compute L*; 2- compute phi
+        - 2nd element: 0 - initialize IGRF field once per year (year.5);
+            n - n is the  frequency (in days) starting on January 1st of each year
+            (i.e. if options(2nd element)=15 then IGRF will be updated on the following days of the
+            year: 1, 15, 30, 45 ...)
+        - 3rd element: resolution to compute L* (0 to 9) where 0 is the recomended value to ensure a
+            good ratio precision/computation time
+            (i.e. an error of ~2% at L=6). The higher the value the better will be the precision, the
+            longer will be the computing time. Generally there is not much improvement for values
+            larger than 4. Note that this parameter defines the integration step (theta)
+            along the field line such as dtheta=(2pi)/(720*[options(3rd element)+1])
+        - 4th element: resolution to compute L* (0 to 9). The higher the value the better will be
+            the precision, the longer will be
+            the computing time. It is recommended to use 0 (usually sufficient) unless L* is not
+            computed on a LEO orbit. For LEO orbit higher values are recommended. Note that this
+            parameter defines the integration step (phi) along the drift shell such as
+            dphi=(2pi)/(25*[options(4th element)+1])
+        - 5th element: allows to select an internal magnetic field model (default is set to IGRF)
+            - 0 = IGRF
+            - 1 = Eccentric tilted dipole
+            - 2 = Jensen&Cain 1960
+            - 3 = GSFC 12/66 updated to 1970
+            - 4 = User-defined model (Default: Centred dipole + uniform [Dungey open model] )
+            - 5 = Centred dipole
+
+    """
+
+    def get_ov(fullov, stind, enind):
+        '''Chop up omni data for multiprocessing'''
```

Comment:
This could be documented better, I have no idea what fullov, stind and enind are, or what the value it returns does.

Reply:
Again old docs, out of scope for this transition.